main.js

```diff
@@ -0,0 +1,82 @@
+import Books from './modules/booksClass.js';
+
+const title = document.getElementById('title');
+const author = document.getElementById('author');
+const add = document.getElementById('buttonAdd');
+const list = document.getElementById('box');
+let currendID = Date.now();
+
+class ListBooks {
+  constructor() {
+    this.arrayOfTitle = [];
+  }
+
+  // Add task to array
+ addTask = (title, author) => {
+   const book = new Books(title, author, currendID);
+   this.arrayOfTitle.push(book);
+ }
+
+  deleteFromLocal = (taskId) => {
+    /* eslint-disable */
+    this.arrayOfTitle = this.arrayOfTitle.filter((task) => task.id != taskId);
```

Comment:
- [ ] Let's try not to disable Linter checks in our codebase as they help is write clean and professional code. Disabling Linters defies the purpose of having it and it's required that we don't disable Linbter checks and that we try to always fix them so we don't have Linter errors in our projects. Kindly fix them 👍🏽